swift/ios-shoppe-demo/Resources/Constants.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  Constants.swift
+//  ios-shoppe-demo
+//
+//  Created on 8/25/20.
+//  Copyright © 2020 FullStory All rights reserved.
+//
+
+
+import Foundation
+
+typealias nib = K.Nib
+typealias cellID = K.CellID
+
+/// Hard coded constants
+struct K {
```

Comment:
Curious, what does `K` stand for?  Or is this an iOS convention?

Reply:
This looks like a mix of conventions. Prefixing a `k` within a constant name is often known as [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation). In Swift, some people will put static constants at the type level. This looks like a mix of both? I haven't seen this before.

Reply:
I meant for K to be `Constants`, its not and iOS thing @nvzqz is right.